### DISCUSSION TITLE
Optimal Distance for a realistic solution for CVRPTW

### DISCUSSION THREAD

**Discussion by manal-syc** [TimefoldAI/timefold-solver#259] [General]
HELLO, for the CVRPTW,
Can we use "Manhattan Distance" or Haversine distance seems to be more effective?
or there is more distance type to calculate a road distance.
thank you.

**Comment by triceo** [comment]
You can use any distance measurement that you can implement. The ones we provide are examples; feel free to modify them to your needs.

**Reply by manal-syc** [reply]
Thank you
